### PR TITLE
Ensure sigs.yaml reflects labels/aliases OWNERS files in k/k

### PR DESCRIPTION
**Pull request by dims**
Fix problems found by `maintainers` script

```
ERROR: file pkg/apis/policy/OWNERS should be in ["sig-apps" "sig-auth"] based on labels/aliases but is in ["sig-auth"]
ERROR: file staging/src/k8s.io/api/policy/OWNERS should be in ["sig-apps" "sig-auth"] based on labels/aliases but is in ["sig-auth"]
ERROR: file staging/src/k8s.io/component-base/OWNERS should be in ["sig-architecture"] based on labels/aliases but is in ["sig-api-machinery"]
ERROR: file staging/src/k8s.io/controller-manager/OWNERS should be in ["sig-api-machinery" "sig-cloud-provider"] based on labels/aliases but is in ["sig-api-machinery"]
```

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by liggitt** [comment]
left one meta-comment about the sigs.yaml / OWNERS label direction

all the changes make sense to me except the cloud provider → controller-manager change

File: sigs.yaml [hunk 1]
```diff
@@ -828,6 +831,7 @@ sigs:
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/cloudprovider/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/controller/cloud/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/src/k8s.io/cloud-provider/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/src/k8s.io/controller-manager/OWNERS
```
[the remark this answers is not in the document]
Comment (dims) [reply sigs.yaml]: Fixing. thanks @liggitt

File: sigs.yaml [hunk 2]
```diff
@@ -225,6 +225,7 @@ sigs:
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/apis/batch/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/apis/core/v1/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/apis/extensions/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/apis/policy/OWNERS
```
Comment (liggitt) [inline sigs.yaml:228]: it might also help to add PodDisruptionBudget to the description above to clarify why this package is in the list

**Comment by liggitt** [comment]
regen sig-apps readme, then lgtm :)

**Comment by dims** [comment]
> regen sig-apps readme, then lgtm :)

ugh! i keep forgetting :) Done. thanks

**Pull request merged**